webapp/src/app.tsx

```diff
@@ -1799,8 +1813,9 @@ ${compileService && compileService.githubCorePackage && compileService.gittag ?
                             {pxt.options.debug && !this.state.running ? <sui.Button key='debugbtn' class='teal' icon="xicon bug" text={"Sim Debug"} onClick={() => this.runSimulator({ debug: true })} /> : ''}
                             {pxt.options.debug ? <sui.Button key='hwdebugbtn' class='teal' icon="xicon chip" text={"Dev Debug"} onClick={() => this.hwDebug()} /> : ''}
                         </div>
-                        <div className="ui editorFloat portrait hide">
-                            <logview.LogView ref="logs" />
+                        <div id="serialPreview" className="ui editorFloat portrait hide">
+                            <logview.LogView ref="simLogs" isSim={true} onClick={() => this.openSerial(true)} />
+                            <logview.LogView ref="devLogs" isSim={false} onClick={() => this.openSerial(false)} />
```

Comment:
This button should only appear when hardware data is available

Reply:
It does.